modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/partials/view-asset-top-common-addons-container.hbs

```diff
@@ -0,0 +1,22 @@
+<!-- Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+-->
+
+<div class="asset-actions pull-right">
+    <span class="asset-show-dependency">
+        <a class="btn btn-default btn-lg active pull-right" href='{{url ""}}/pages/impact?path={{path}}' target="_blank">
+            Show Dependency
```

Comment:
This "t helper internationalization" has been discussed with the G-Reg team at the reviews and since the current implementation does not include "18nResources.json" like in API-Manager, we agreed to show the String as plain text, like the examples mentioned below.

Eg:
1) https://github.com/wso2/product-greg/blob/master/modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/themes/default/partials/view-asset-top-common-addons-container.hbs

2) https://github.com/wso2/product-greg/blob/master/modules/es-extensions/publisher/asset/endpoint/themes/default/partials/view-asset-details-container.hbs